osu.Game/Skinning/LegacySkin.cs

```diff
@@ -63,7 +68,12 @@ protected override void Dispose(bool isDisposing)
                     switch (global)
                     {
                         case GlobalSkinConfiguration.ComboColours:
-                            return SkinUtils.As<TValue>(new Bindable<List<Color4>>(Configuration.ComboColours));
+                            if (Configuration.ComboColours.Any())
```

Comment:
you've bypassed `Configuration`. configuration is public. now you can get two values for combo colours depending on how you access.

Reply:
I fail to understand what you mean, care to explain further?

Reply:
`skin.Configuration.ComboColours` is going to return different results to `skin.GetConfig(ComboColours)`

Reply:
I'm not sure if this is causing issues though? if the combo colours retrieved is null (in DHO), the white colour will be used. what I'm trying to say is that the correct way to use this is by using a `SkinProvidingContainer`.

Reply:
Having two ways of accessing the same value (or at least what *looks like* it should be the same value, as this does) return two different results is bug-prone by default and a violation of encapsulation. We don't want to have to remember in the future which way of accessing it is *right*. There should be only one way of accessing a particular value.

Reply:
are you talking about `skin.Configuration.ComboColours` returning empty list and `GetConfig(ComboColours` returning null?